android/build.gradle

```diff
@@ -27,10 +27,10 @@ repositories {
 }
 
 android {
-    compileSdk 31
+    compileSdk 33
     defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 31
+        minSdkVersion 16
+        targetSdkVersion 33
```

Comment:
Yeah, you’re right. It’d surely be better to do these version changes in another PR but I needed the target sdk to be at least `33` to access one of the new PiP parameters: `setExpandedAspectRatio`. Then I took the opportunity also to fix the min sdk that was set to `21` but should be `16`, which is currently the min version supported by the Android Player SDK and hasn't been fixed until now.

From now on, I’ll leave these version changes in different PRs since this case was pretty much an exception.
Thanks for the feedback 🙂